lib/CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ set(LIBFACTER_COMMON_SOURCES
     "src/facts/external/yaml_resolver.cc"
     "src/facts/map_value.cc"
     "src/facts/resolver.cc"
+    "src/facts/resolvers/augeasversion_resolver.cc"
```

Comment:
Yeah, that might be useful to have the "major, minor, full" etc properties we do for other "version" facts.